health/notifications/alarm-notify.sh.in

```diff
@@ -1703,6 +1713,55 @@ send_awssns() {
 	return 1
 }
 
+# -----------------------------------------------------------------------------
+# Matrix sender
+
+send_matrix() {
+    local homeserver="${1}" webhook accesstoken rooms="${2}" httpcode sent=0 payload color
+
+	[ "${SEND_MATRIX}" != "YES" ] && return 1
+
+    # check if access token was provided
+    if [ "${SEND_MATRIX}" == "YES"  ] && [ -n "${MATRIX_ACCESSTOKEN}" ]; then
+        accesstoken="${MATRIX_ACCESSTOKEN}"
```

Comment:
few minor things:
 - unused `color` var
 - `[ "${SEND_MATRIX}" == "YES"  ]` is not needed because

> [ "${SEND_MATRIX}" != "YES" ] && return 1

Reply:
could be?
```

        [ "${SEND_MATRIX}" != "YES" ] && return 1
        [ -z "${MATRIX_ACCESSTOKEN}" ] && return 1

        accesstoken="${MATRIX_ACCESSTOKEN}"
```